caluma_interval/interval.py

```diff
@@ -61,15 +71,41 @@ def start_case(self, form):
         self.action_count += 1
 
     @staticmethod
-    def parse_interval(interval):
-        start = None
+    def parse_startdate(value):
+        try:
+            return parse_date(value)
+        except isoerror.ISO8601Error:
+            return False
+
+    @staticmethod
+    def parse_duration(value):
+        try:
+            if not re.match(ISO8601_PERIOD_DATE_REGEX, value):
+                raise isoerror.ISO8601Error
+            return parse_duration(value)
+        except isoerror.ISO8601Error:
```

Comment:
You're catching the exception from two lines above. Could just `return False` there instead, no?

Reply:
Nice catch!